### PR TITLE
Active admin

### PR DESCRIPTION
**Pull request by akitaonrails**
Basic Active Admin configuration

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -65,10 +55,14 @@ def current_team
 
   def after_sign_in_path_for(resource)
     if params.dig(:user, :reset_password_token)
-      session[:current_team_slug] = current_user.teams.first.slug
+      session[:current_team_slug] = current_user.try(:teams).try(:first).try(:slug)
```
Comment (daniloisr) [inline app/controllers/application_controller.rb]: What you think about using Safe Navigation Operator?

```ruby
current_user&.teams&.first&.slug
```

File: app/models/admin_user.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class AdminUser < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :trackable, :validatable
+
+  validates :email, presence: true
```
Comment (daniloisr) [inline app/models/admin_user.rb]: We can configure Active Admin to be accessed by a `User` model, what is the reason to use this new model `AdminUser`?

File: app/models/admin_user.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class AdminUser < ActiveRecord::Base
```
Comment (daniloisr) [inline app/models/admin_user.rb]: We can use the model `User` to access Active Admin, any reason to use this one?

Comment (akitaonrails) [inline app/models/admin_user.rb]: Yep, I prefer the Super Admin to be completely separated from the normal User. I don't want to add a flag, roles or anything. A Team Administrator is a User with an Enrollment to a Team and a is_admin flag there. But a Super Admin has no ties to any Team and he can only log in to Active Admin.

Comment (daniloisr) [inline app/models/admin_user.rb]: Got it, thanks :v:

**Pull request merged**